install/kubernetes/cilium/templates/hubble-ui/ingress.yaml

```
@@ -25,7 +26,7 @@ spec:
     - host: {{ . }}
       http:
         paths:
-          - path: /
+          - path: {{ $baseUrl }}
```

I thihk it would be safer to

```suggestion
          - path: {{ $baseUrl | quote }}
```